Sources/Stripe/Model.swift

```diff
@@ -249,7 +249,7 @@ public struct Invoice: Codable, Equatable {
   public var customer: Customer.Id
   public var discount: Discount?
   public var id: Id?
-  public var invoicePdf: String
+  public var invoicePdf: String?
```

Comment:
While working on this branch I noticed that upcoming invoices weren't decoding because this field can be missing/null.